mani_skill/envs/tasks/pour_bowl_contents.py

```diff
@@ -0,0 +1,124 @@
+from pathlib import Path
+from collections import OrderedDict
+from typing import Dict, Union
+
+import numpy as np
+import torch
+import torch.random
+from transforms3d.euler import euler2quat
+import sapien
+import sapien.physx as physx
+import sapien.render
+import transforms3d
+
+from mani_skill.agents.robots import Fetch, Panda, Xmate3Robotiq
+from mani_skill.envs.sapien_env import BaseEnv
+from mani_skill.sensors.camera import CameraConfig
+from mani_skill.utils.building import actors
+from mani_skill.utils.geometry import rotation_conversions
+from mani_skill.utils.registration import register_env
+from mani_skill.utils.sapien_utils import look_at
+from mani_skill.utils.scene_builder.table.table_scene_builder import TableSceneBuilder
+from mani_skill.utils.structs.pose import Pose
+from mani_skill.utils.structs.types import Array
```

Comment:
make sure not to import unused packages (I realize some of other code does this and will fix that).